pt_miniscreen/demo_projects/pi_top_4/pong_game/project.cfg

```diff
@@ -1,3 +1,4 @@
 [project]
 title=Pong Game
 start=python3 -B pong.py
+exit_condition=POWER_BUTTON_PRESS
```

Comment:
Press, then if someone is holding because they can't remember how to turn off the project it exits before the pi-top powers down. It will still work with a flick as well :P. The PR description is a little out of date.